Strings/max_length_concatenated_string_unique_characters.py

```diff
@@ -0,0 +1,21 @@
+class Solution:
```

Comment:
The code looks good, could you also add description and question with sample io on top of file? Thanks